docs/UserGuide.md

```diff
@@ -112,14 +112,28 @@ Examples:
 * `create t/dept software t/dept marketing`
 
 An example of the `create` command in action:
-[Create](images/create.png)
+![Create](images/create.png)
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds a person to JABPro.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/[CATEGORY] TAGNAME]…​`
 
+**Notes regarding the design of the `add` command:**
+* The uniqueness of the person is determined by the name only. This means that you cannot have 2 persons with the same name in the application book.
+* All other fields other than name can be identical between different people in JABPro.
+* `n/NAME` - `Name` must be alphanumeric (Letters and numbers, no symbols allowed such as `/`, `,` ...)
```

Comment:
Minor nitpick but maybe change to `NAME` instead of `Name`

Reply:
Will change in next push